notebooks/official/prediction/custom_batch_prediction_feature_filter.ipynb

```diff
@@ -0,0 +1,1586 @@
+        "\n",
+        "The `run` function creates a training pipeline that trains and creates a `Model` object. After the training pipeline completes, the `run` function returns the `Model` object."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "mxIxvDdglugx"
+      },
+      "outputs": [],
+      "source": [
+        "job = aiplatform.CustomTrainingJob(\n",
+        "    display_name=JOB_NAME,\n",
+        "    script_path=\"task.py\",\n",
+        "    container_uri=TRAIN_IMAGE,\n",
+        "    requirements=[\"google-cloud-bigquery>=2.20.0\", \"db-dtypes\"],\n",
+        "    model_serving_container_image_uri=DEPLOY_IMAGE,\n",
+        ")\n",
+        "\n",
+        "MODEL_DISPLAY_NAME = \"penguins-unique\"\n",
+        "\n",
+        "# Start the training\n",
+        "if TRAIN_GPU:\n",
+        "    model = job.run(\n",
+        "        dataset=dataset,\n",
+        "        model_display_name=MODEL_DISPLAY_NAME,\n",
+        "        bigquery_destination=f\"bq://{PROJECT_ID}\",\n",
+        "        args=CMDARGS,\n",
+        "        replica_count=1,\n",
+        "        machine_type=TRAIN_COMPUTE,\n",
+        "        accelerator_type=TRAIN_GPU.name,\n",
+        "        accelerator_count=TRAIN_NGPU,\n",
+        "    )\n",
+        "else:\n",
+        "    model = job.run(\n",
+        "        dataset=dataset,\n",
+        "        model_display_name=MODEL_DISPLAY_NAME,\n",
+        "        bigquery_destination=f\"bq://{PROJECT_ID}\",\n",
+        "        args=CMDARGS,\n",
+        "        replica_count=1,\n",
+        "        machine_type=TRAIN_COMPUTE,\n",
+        "        accelerator_count=0,\n",
+        "    )"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "make_prediction"
+      },
+      "source": [
+        "## Send Batch Prediction job request with filtered features (instanceConfig field)\n",
+        "\n",
+        "Now that the model is ready, you can send batch prediction request directly from the model resource without needing to deploy the model to an endpoint.  For data types that support both batch and online predictions, use batch predictions when you don't require an immediate response and want to process accumulated data by using a single request. \n",
+        "\n",
+        "There are many times that the input data you have for batch prediction need some preprocessing before the predictor container can actually take the request. The most common preprocessing heard are feature filtering and instance container transforming. The feature filtering removes instance identifiers or input metadata that is stored in the dataset for various reasons but not for the predictors. You can filter and/or transform your batch input by specifying `instanceConfig` in your `BatchPredictionJob` request (**v1beta1 only**).\n",
```

Comment:
This paragraph repeats a lot of the information in the docs. To adhere to DRY (don't repeat yourself), consider a link to docs instead, and focusing on the information specific to this notebook.

> The most common preprocessing heard are feature filtering and instance container transforming.

Not sure what "instance container" is referring to. I thought the transform lets you send the instances to your prediction container in either a JSON array or object.

Consider removing "tranformation" because it's not covered in this notebook. maybe something like:

Sometimes, your input data does not match the data format that the predictor accepts. Feature filtering lets you either exclude certain fields (such as identifiers or metadata) that are in the input data from your prediction request, or include only a subset of fields from the input data in your prediction request, without having to do any custom pre/post-processing in the prediction container.